content/docs/command-reference/exp/branch.md

```diff
@@ -15,26 +15,28 @@ positional arguments:
 
 ## Description
 
-Makes a named Git
-[`branch`](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-containing the target `experiment` (making it
-[persistent](/doc/user-guide/experiment-management#persistent-experiments)). For
-[checkpoint experiments](/doc/command-reference/exp/run#checkpoints), the new
-branch will contain multiple commits (the checkpoints).
+Makes a given Git [`branch`] containing the target `experiment`. This makes the
+experiment into a [regular commit], or several in the case of [checkpoint
+experiments] (one per checkpoint).
```

Comment:
@iesahin here's that rephrasing I mentioned. Checkpoints -> multiple commits info is preserved.

Please also notice how I changed the term "persistent" into "regular commit" as suggested in https://github.com/iterative/dvc.org/issues/2960. WDYT? (Unrelated to this PR though)

Reply:
"regular commit" is very good.